### ISSUE TITLE
Only login shortcode works

### ISSUE THREAD

**Issue by uklertis** [dalenguyen/firebase-wordpress-plugin#194]
Hello, I tried this plugin and it's looking cool. I'm currently testing and then I'll probably puy pro and woocommerce versions. But when I use shortcodes, only the login shortcode works. 
I'm using it like so: 
<img width="297" alt="image" src="https://user-images.githubusercontent.com/29019100/160554203-758ca79e-1d7e-455a-83e9-b3139fa57947.png">

When I try [these shortcodes](https://firebase-wordpress-docs.readthedocs.io/en/latest/intro/shortcode-list.html), none of them work. Only login works. Do I need to buy the pro version to use them?

**Comment by dalenguyen** [comment]
Yes @uklertis, the other shortcodes are only available in the PRO version.

**Comment by uklertis** [comment]
So the free version is more like a demo version. I see. Thank you
